core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/cluster/provision/clusterproxy/ClusterProxyService.java

```diff
@@ -93,7 +110,7 @@ private ClusterServiceConfig serviceConfig(Stack stack) {
     private String knoxUrl(Stack stack) {
         String gatewayIp = stack.getPrimaryGatewayInstance().getPublicIpWrapper();
         Cluster cluster = stack.getCluster();
-        return String.format("https://%s/%s", gatewayIp, cluster.getGateway().getPath());
+        return String.format("https://%s:9443/%s", gatewayIp, cluster.getGateway().getPath());
```

Comment:
I don't think this port should be hard-coded.

Reply:
this should come from stack.cluster.gateway.gatewayPort

Reply:
Reverted this change.